### PR TITLE
LPS-195643 JournalFeedTypeUpgradeProcess: DuplicateVocabularyException: A category vocabulary with the name tipo already exists

### PR DESCRIPTION
**Pull request by georgel-pop-lr**
For steps and more details, please check https://liferay.atlassian.net/browse/LPS-195643

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/v4_4_4/JournalFeedTypeUpgradeProcess.java [hunk 1]
```diff
@@ -208,9 +208,22 @@ private void _addJournalFeedTypesAndAssetEntries() throws Exception {
...
+							UpgradeProcessUtil.getDefaultLanguageId(
+								company.getCompanyId()));
+
+						Map<Locale, String> assetVocabularyTitleMap =
+							_localization.getLocalizationMap(
+								_language.getAvailableLocales(
+									company.getGroupId()),
+								defaultLocale, "type");
+
+						String assetVocabularyName =
```
Comment (jkappler) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/v4_4_4/JournalFeedTypeUpgradeProcess.java:221]: Hi @georgel-pop-lr, all looks good. But can we rename this variable to assetVocabularyTitle so as to match the map?

Thanks!

Comment (georgel-pop-lr) [reply modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/v4_4_4/JournalFeedTypeUpgradeProcess.java:221]: Yes sure!

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
thanks @georgel-pop-lr ❤️!

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**